### ISSUE TITLE
guid node id => server-example.py

### ISSUE THREAD

**Issue by AndreasHeine** [FreeOpcUa/python-opcua#1006]
Hey,

looks like something went wrong with the latest update on guid node id in the server-example.py:
![Unbenannt](https://user-images.githubusercontent.com/56362817/80197876-94fdf180-861f-11ea-8c4a-9d7368e57c91.PNG)

python 3.7.3 32bit
opcua 0.98.9

**Comment by AndreasHeine** [comment]
@Michael-List

**Comment by zerox1212** [comment]
There was some work on GUID recently. I guess `NodeId()` expects a string instead of a UUID object. Probably needs PR and review as we decided the to use UUID objects in python and never GUID as a string or bytes.

**Comment by Michael-List** [comment]
@AndreasHeine I will take a look at it

**Comment by Michael-List** [comment]
@AndreasHeine I could identify the problem.

- Version 0.98.9 was released on 18.12.19.
- On 14.03.20 I spotted a bug with guid node id that was fixed in this pull request: https://github.com/FreeOpcUa/python-opcua/pull/975/files
- In this pull request I also added a guid node id example to the server-example.py because there was none.

If you are now using the newest server-example.py file with the lib version 0.98.9 my pull request is missing. Thats why this error is happening. Either you comment out my guid node id example if you don't need it or you install the python lib directly from github so that you are getting the newest version.
